nodejs/week1/homework/warmup.js

```diff
@@ -0,0 +1,20 @@
+console.log("inside warmup file");
+class Circle {
+    constructor(radius) {
+        this.radius = radius;
+    }
+    getDiameter() {
+        console.log(this.radius * 2);
```

Comment:
```suggestion
const diameter = 2 * this.radius
        console.log(diameter)
 return diameter
```
the function should return a value in this case. same applies to the other functions